src/surge-xt/gui/widgets/PatchSelector.cpp

```diff
@@ -639,9 +639,9 @@ void PatchSelector::showClassicMenu(bool single_category, bool userOnly)
         // get just the category name and not the path leading to it
         std::string menuName = storage->patch_category[rightMouseCategory].name;
 
-        if (menuName.find_last_of(PATH_SEPARATOR) != std::string::npos)
+        if (menuName.find_last_of("\\/") != std::string::npos)
```

Comment:
This is odd why are you testing for both slashes

Reply:
I noticed in FXP there can be either slash used depending on how the user typed the category string.

I didn't see if this is being normalized anywhere...?